### PR TITLE
Make the docs clearer about models

### PR DESCRIPTION
**Pull request by GuilhemN**

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -168,19 +168,29 @@ Use models
 As shown in the example above, the bundle provides the ``@Model`` annotation.
 When you use it, the bundle will deduce your model properties.
 
+.. note::
+
+    A model can be a form type, an entity or any object.
```
Comment (dbu) [inline Resources/doc/index.rst]: We could be more specific while we are at it. ... a Symfony form type, a Doctrine ORM entity or a general PHP object.

(Am i correct assuming that the doctrine odm annotations are not used?)

File: Resources/doc/index.rst [hunk 2]
```diff
@@ -191,8 +201,8 @@ It has two options:
 
         /**
          * @SWG\Response(
-         *   response="200",
-         *   description="Success",
+         *   response="200"
```
Comment (dbu) [inline Resources/doc/index.rst]: Isn't the comma needed here?

File: Resources/doc/index.rst (hunk 1 above)
Comment (GuilhemN) [reply Resources/doc/index.rst]: > We could be more specific while we are at it. ... a Symfony form type, a Doctrine ORM entity or a general PHP object.

+1, updated

> (Am i correct assuming that the doctrine odm annotations are not used?)

 You mean ``Doctrine orm annotation``? The extractor is not easy to find but they are extracted, see https://github.com/symfony/doctrine-bridge/blob/master/PropertyInfo/DoctrineExtractor.php.

File: Resources/doc/index.rst (hunk 2 above)
Comment (GuilhemN) [reply Resources/doc/index.rst]: indeed :)

**Comment by dbu** [comment]
Thanks.

The bridge extractor phpdoc talks about odm metadata, but i dont think it looks at any of the odm's. Thus it seems correct to only talk about orm.

**Comment by GuilhemN** [comment]
Ok let's keep your reword as is then :)

**Pull request merged**